source/utilities.cc

```diff
@@ -76,50 +76,47 @@ bool create_directory(const std::string &name)
   std::string name_cleaned = name;
   name_cleaned.erase(std::remove(name_cleaned.begin(),name_cleaned.end(),' '),name_cleaned.end());
 #ifdef DEAL_II_SAK_WITH_BOOST
-  return boost::filesystem::create_directories(name_cleaned + "/");
+  boost::filesystem::create_directories("./" + name_cleaned + "/");
 #else
   std::string cmd = "";
   cmd = "mkdir -p " + name_cleaned + "/";
-  if ( int( std::system( cmd.c_str() ) == 0 ) )
-    {
-      return true;
-    }
-  else
-    {
-      return false;
-    }
 #endif
+  return file_exists(name_cleaned + "/");
```

Comment:
It should not work because I made a big mistake: the output of *std::system* seems to not be related to the success of the unix command contrary to what I believed...